mongo/integration/unified/collection_operation_execution.go

```diff
@@ -57,7 +57,14 @@ func executeAggregate(ctx context.Context, operation *operation) (*operationResu
 			}
 			opts.SetCollation(collation)
 		case "comment":
-			opts.SetComment(val.StringValue())
+			// TODO(GODRIVER-2386): when document support for comments is added, we can replace this switch condition
+			// TODO with `opts.SetComment(val)`
```

Comment:
```suggestion
			// with `opts.SetComment(val)`
```

Reply:
This is [better comments](https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments) practice, where you keep the annotation constant on each line of the comment for highlighting.  But if we feel strongly as a team that we don't want to do this, I can remove it. 

<img width="1041" alt="Screen Shot 2022-04-26 at 10 50 03 AM" src="https://user-images.githubusercontent.com/24281431/165352066-774d5819-a13b-4b19-a344-edd3cff38cde.png">

Reply:
Ah I'd never used the better comments extension, cool! I don't feel strongly that we should _not_ do this, but there's no precedence in the driver currently. If you'd like to make a separate ticket/PR to update all `TODO` statements to use this format, I think that'd be good. Feel free to leave this one as-is if you want.

Reply:
Sounds good, here's the PR for that: https://github.com/mongodb/mongo-go-driver/pull/924